recipes/libpcsclite/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "libpcsclite" %}
+{% set version = "1.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LudovicRousseau/PCSC/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 856a1daed9a5ae1ac265c75ce9f81743e66a972e5be5a1d3a2196ed23f75ea26
+
+build:
+  skip: True  # [win]
+  script:
+    - ./bootstrap
+    - ./configure  # [linux]
+    - ./configure --disable-libsystemd  # [osx]
+    - make
+    # Test is here
+    - make distcheck  # [linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ cdt('systemd-devel') }}  # [linux]
+    - autoconf-archive
+    - automake
+    - flex
+    - libtool
+    - libudev  # [linux]
+    - libudev-devel-cos6-x86_64  # [not linux]
+    - libusb
+    - make
+    - pkg-config 
+    - sysroot_linux-64 2.17  # [linux64]
+  host:
+  run:
+
+test:
+  commands:
+    # Placeholder
+    - ls
+    - ls ..
```

Comment:
This section should have existence tests for each of the artifact types: config files, libraries, headers, binaries.

Binaries should be tested by calling them with `--help` or `--version`.

Reply:
I had to run the test suite in the build section because I can't figure out how do to so otherwise, any pointers how I might proceed?

Reply:
just add this
```suggestion
    - pcscd --version
```

Reply:
You can run the test suite during the build phase (or not at all). The test phase of this recipe is for packaging tests (check that the artifacts are packaged correctly) that's why we want existence tests and checks that the CLIs of any binaries work.

Reply:
I'll do the CLI test that @leofang suggested thanks!

Reply:
@carterbox if you have other suggestions please let me know as I'm kinda clueless...